docs/spec/v1/kustomizations.md

```diff
@@ -1649,6 +1649,69 @@ secretGenerator:
       - .dockerconfigjson=ghcr.dockerconfigjson.encrypted
 ```
 
+### SOPS Encrypted Kustomize patches
+
+SOPS-encrypted data can be stored as [Kustomize `patches`](https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/patches/) as long as they're in separate files, not inlined in `kustomization.yaml`. The kustomize-controller decrypts these before executing kustomization pipeline, allowing for adding secret data to resources or merging Secrets. For example:
+
+```yaml
+# patch1.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret
+stringData:
+  secretConfig: "my-secret-configuration"
+```
+
+```yaml
+# patch2.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret
+stringData:
+  secretToken: "my-secret-token"
+```
+
+```yaml
+# base.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret
+stringData:
+  publicConifg: "my-public-config"
```

Comment:
I don't think Flux docs should tell users to leave Secrets in plain text in the repo. I think the base Secret should contain no keys.

Reply:
Does that apply to the "Kustomize secretGenerator" chapter as well?

Reply:
Sorry, put the comment in the wrong place...  My question was directed at this comment.
> A better place for this documentation is here: https://fluxcd.io/flux/guides/mozilla-sops/
@nagygergo please copy the section from this PR into https://github.com/fluxcd/website/edit/main/content/en/flux/guides/mozilla-sops.md